src/components/shared/OfficeItem.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ItemDescription from "./ItemDescription";
+import Avatar from "./Avatar";
+import styled from "styled-components";
+
+const renderGrid = {
+  wrapperClass: "card",
+  imgClass: "card-img-top rounded-circle",
+  bodyClass: "card-body",
+  titleClass: "card-title"
+};
+
+const renderList = {
+  wrapperClass: "media",
+  imgClass: "cover-image rounded-circle",
+  bodyClass: "media-body",
+  titleClass: "mt-0"
+};
+
+const Truncate = styled.div`
+  p {
+    width: ${props => (props.layout === "list" ? "85vw" : "93%")};
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+const OfficeItem = ({ office, layout }) => {
+  const renderStyle = layout === "list" ? renderList : renderGrid;
```

Comment:
Reuse LayoutTypes constant defined above

Reply:
Not sure what I should do...?

Reply:
done.